src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package duke;
+
+public class DukeException extends Exception{
+    public DukeException(String errorMessage){
```

Comment:
Some of your methods are missing spacings before the ```{```. 
```suggestion
public class DukeException extends Exception {
    public DukeException(String errorMessage) {
```
Perhaps you might want to look at the same issue in several other places too